staging/src/k8s.io/client-go/transport/spdy/spdy.go

```diff
@@ -48,6 +48,7 @@ func RoundTripperFor(config *restclient.Config) (http.RoundTripper, Upgrader, er
 		Proxier:    proxy,
```

Comment:
should DialContext be added to RoundTripperConfig so we don't have to fix up the returned upgradeRoundTripper after the fact?

Reply:
sgtm